recipes/pistache/all/conanfile.py

```diff
@@ -0,0 +1,104 @@
+import os
+import glob
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+required_conan_version = ">=1.29.1"
+
+
+class PistacheConan(ConanFile):
+    name = "pistache"
+    license = "Apache-2.0"
+    homepage = "https://github.com/pistacheio/pistache"
+    url = "https://github.com/conan-io/conan-center-index"
+    topics = ("http", "rest", "framework", "networking")
+    description = "Pistache is a modern and elegant HTTP and REST framework for C++"
+    settings = "os", "compiler", "arch", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        compilers = {
+            "gcc": "5",
+            "clang": "3.4",
+        }
+        if self.settings.os != "Linux":
+            raise ConanInvalidConfiguration("Pistache is only support by Linux.")
+
+        # https://github.com/pistacheio/pistache/issues/835
+        if self.settings.compiler == "clang" and self.settings.compiler.libcxx == "libc++":
+            raise ConanInvalidConfiguration("Pistache can not use libc++. For more info see pistacheio/pistache#835.")
```

Comment:
Reading that issue, it looks like pistache needs c++17
Maybe filtering on that c++ standard makes it clearer?
Because I think newer clang's are able to build pistache.

Reply:
Yes, but they have to remove the internal string_view and add c++17 as requirement. For now, even clang 9 is not able to build Pistache with libc++ https://github.com/conan-io/conan-center-index/pull/3719#issuecomment-734956306

We could patch it, like removing ambiguous files

Reply:
Done, I've patched to use the standard string_view, of course, it requires C++17